### PR TITLE
de 1091 facebook please reduce the amount of

### PR DESCRIPTION
**Pull request by paddie**
# Description of change
(write a short description or paste a link to JIRA)

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_facebook/streams.py [hunk 1]
```diff
@@ -112,12 +114,35 @@ def process_account(
...
+                        job = cast(AdReportRun, async_job.api_get())
+
+                        pct: int = job["async_percent_completion"]
+                        status: str = job["async_status"]
+
+                        logger.info(f"{status}: {pct}%")
+
+                        # https://developers.facebook.com/docs/marketing-api/insights/best-practices/#asynchronous
+                        # both fields need to be set to signify completion
+                        if status == "Job Completed" and pct == 100:
```
Comment (wizhi) [inline tap_facebook/streams.py:138]: This is really funky lol.

File: main.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from tap_facebook import main
+
+if __name__ == "__main__":
+    main()
```
Comment (wizhi) [inline main.py:4]: If we do want to check these into git, maybe we should include the typical `.vscode/launch.settings` or whatever it's called as well, for debugging.

**Review by wizhi — APPROVED** [review]
I have to admit, I don't see _why_ this should solve the issue, since turning the job asynchronous still results in the same amount of eventual data Facebook has to return.

I'm going to assume, that the sync/async internals amount to the same thing, but just expose different interfaces for us to use.

File: main.py (hunk 2 above)
Comment (paddie) [reply main.py:4]: yeah, I actually wanted to but the .vscode folder was ignored.. I agree though.

Comment (paddie) [reply main.py:4]: un-ignoring it now :)

File: tap_facebook/streams.py (hunk 1 above)
Comment (paddie) [reply tap_facebook/streams.py:138]: yeah, which is why I added the documentation reference :)

**Pull request merged**